app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SearchResultViewModel.java

```diff
@@ -136,23 +135,41 @@ public boolean getIsMySession() {
         return isMySession;
     }
 
+    public int getSearchResultId() {
+        return searchResultId;
+    }
+
+
     static SearchResultViewModel createTitleType(@NonNull Session session, Context context,
             MySessionsRepository mySessionsRepository) {
-        return new SearchResultViewModel(session.title, R.drawable.ic_title_12_vector,
-                R.string.title, session, context, mySessionsRepository);
+        return new SearchResultViewModel(session.title, Type.TITLE, session, context, mySessionsRepository);
     }
 
     static SearchResultViewModel createDescriptionType(@NonNull Session session, Context context,
             MySessionsRepository mySessionsRepository) {
-        return new SearchResultViewModel(session.desc, R.drawable.ic_description12_vector,
-                R.string.description, session, context, mySessionsRepository);
+        return new SearchResultViewModel(session.desc, Type.DESCRIPTION, session, context, mySessionsRepository);
     }
 
     static SearchResultViewModel createSpeakerType(@NonNull Session session, Context context,
             MySessionsRepository mySessionsRepository) {
-        return new SearchResultViewModel(session.speaker.name, R.drawable.ic_speaker_12_vector,
-                R.string.speaker, session, context, mySessionsRepository);
+        return new SearchResultViewModel(session.speaker.name, Type.SPEAKER, session, context, mySessionsRepository);
     }
 
+    public enum Type {
+        TITLE(R.drawable.ic_title_12_vector),
+        DESCRIPTION(R.drawable.ic_description12_vector),
+        SPEAKER(R.drawable.ic_speaker_12_vector),
+        ;
```

Comment:
Is this intentional?
Or is it better? `SPEAKER(R.drawable.ic_speaker_12_vector);`